DOCS.md

```diff
@@ -1336,11 +1337,46 @@ __Arguments__
 
 ---------------------------------------
 
+<a name="markAsDelivered"></a>
+### api.markAsDelivered(threadID, messageID[, callback]])
+
+Given a threadID and a messageID will mark that message as delivered. If a message is marked as delivered that tells facebook servers that it was recieved.
+
+You can also mark new messages as delivered automatically. See [api.setOptions](#setOptions).
```

Comment:
Should we mention that this is on by default?

Reply:
Yes that's a good idea.